### ISSUE TITLE
Loading from list of filenames should return a list

### ISSUE THREAD

**Issue by samvasko** [uiri/toml#78]
Related to [this section](https://github.com/uiri/toml/blob/master/toml.py#L41-L48) of code.
When calling `.loads([])` my expectation would be to get a list, not a dictionary with all the values merged together. I do not really see a use case for the latter. It also causes unexpected rewrites. 

Example:
```Python
with open('file1.toml', 'w') as f:
    f.write('a = 1\n')

with open('file2.toml', 'w') as f:
    f.write('a = 2\n')

tom = toml.load(['file1.toml', 'file2.toml']);
print(tom) # Prints {'a': 2} 
```

**Comment by RazerM** [comment]
This is used like [`ConfigParser.read`](https://docs.python.org/3.6/library/configparser.html#configparser.ConfigParser.read), usually so that a local config file can take precedence over a file in `$HOME` and/or elsewhere.

**Comment by uiri** [comment]
If you want a list, why not use a list comprehension?

```python
tomlified_filelist = [toml.load(f) for f in filelist]
```

I'm hesitant to change the behaviour because although you do not see a use case for it, someone downstream may be depending upon its behaviour and I would rather not break dependency that without a good reason.

**Comment by samvasko** [comment]
I guess this is more of documentation issue then 📖

**Comment by uiri** [comment]
Documentation Fixed by #79 

Thanks Sam!
